client/packages/requisitions/src/ResponseRequisition/api/operations.generated.ts

```diff
@@ -797,34 +832,30 @@ export type CustomerProgramSettingsFragment = {
         __typename: 'PeriodNode';
         id: string;
         name: string;
+        startDate: string;
+        endDate: string;
       }>;
     }>;
   }>;
 };
 
-export type CustomerProgramSettingsQueryVariables = Types.Exact<{
+export type ProgramRequisitionSettingsByCustomerQueryVariables = Types.Exact<{
+  customerStoreId: Types.Scalars['String']['input'];
   storeId: Types.Scalars['String']['input'];
 }>;
 
-export type CustomerProgramSettingsQuery = {
+export type ProgramRequisitionSettingsByCustomerQuery = {
   __typename: 'Queries';
-  customerProgramRequisitionSettings: Array<{
+  programRequisitionSettingsByCustomer: {
     __typename: 'CustomerProgramRequisitionSettingNode';
-    programName: string;
-    tagName: string;
-    programId: string;
-    customerAndOrderTypes: Array<{
-      __typename: 'CustomerAndOrderTypeNode';
-      customer: {
-        __typename: 'NameNode';
-        code: string;
-        id: string;
-        isCustomer: boolean;
-        isSupplier: boolean;
-        isOnHold: boolean;
-        name: string;
-        store?: { __typename: 'StoreNode'; id: string; code: string } | null;
-      };
```

Comment:
The main reason we couldn't use this structure anymore is because previously we were implicitly filtering what programs to query by 'store_id' which was the suppliers store id (ie the supplying store had to have the name tag of the program).

We want to find programs available to customers which aren't necessarily available to the supplier, which would mean querying every program and every customer and every order type and every period which is unwieldy!

This is why we refactored to select a customer first, and then make a filtered query based on this selection.